modules/ting_search_ortograf/COPYING

```diff
@@ -0,0 +1,33 @@
+© DBC A/S and TING.Community
+
+Ding.TING software.
+
+This Drupal and PHP based software is created by developers employed
+or contracted at DBC A/S or TING Library Partners.
+
+Drupal are licensed under the GNU General Public License, version 2 or
+later. That means you are free to download, reuse, modify, and
+distribute any files hosted in Drupal.org's Git repositories under the
+terms of either the GPL version 2 or version 3, and to run Drupal in
+combination with any code, with any license that is compatible with
+either versions 2 or 3, such as the Affero General Public License
+(AGPL) version 3.
+
+This software is licensed under GNU General Public License, version 2 or later.
+Changes in this code run or offered by a third party should be offered
+to TING.Community – ting.dk.
+
+The modules, components and SW in general are developed for use in
+cultural sector. Ding.TING was originally developed upon Drupal 6 and
+PHP as an element in TING by Aarhus and Copenhagen City Libraries.
+This version is totally reengineered and mainly developed by DBC A/S.
+The GPL license doesn't repeal the intellectual properties which
+belonges to those TING partners and the legal person who will enforce
+the rights is DBC on behalf of the TING.Community.
+
+DBC A/S (dbc.dk) is a public owned company working with bibliographic
+services and software development for cultural sector, especially the
+libraries.
+
+TING.Community is a community defined as "An open eco-system for
+digital [cultural] cooperation, sharing and development."
```

Comment:
I do not think this file is needed. We already provide a `COPYING` in the root of our repository.